rapidtide/util.py

```diff
@@ -28,13 +29,16 @@
 import matplotlib.pyplot as plt
 import numpy as np
 import pyfftw
+import pyfftw.interfaces.scipy_fftpack as fftpack
 from numba import jit
-from scipy import fftpack
 
 import rapidtide._version as tide_versioneer
 import rapidtide.io as tide_io
 
-fftpack = pyfftw.interfaces.scipy_fftpack
```

Comment:
I think this is a cleaner import format for `fftpack`.

Reply:
Seems to be!